src/vs/workbench/contrib/terminalContrib/quickFix/browser/quickFixAddon.ts

```diff
@@ -283,7 +283,7 @@ export class TerminalQuickFixAddon extends Disposable implements ITerminalAddon,
 			updateLayout(this._configurationService, e);
 			this._audioCueService.playAudioCue(AudioCue.terminalQuickFix);
 
-			const parentElement = e.parentElement?.parentElement?.parentElement?.parentElement;
+			const parentElement = e.closest('.xterm.terminal.focus') as HTMLElement;
```

Comment:
Could this just be `.xterm`?

Reply:
Yeah it could, but I thought there was no harm in being more specific

Reply:
Less specific should actually be less likely to break. Also `.xterm` is safe as it should never change in xterm.js and there should never be nested xterm terminals